include/spirv-tools/libspirv.hpp

```diff
@@ -300,10 +307,19 @@ class SpirvTools {
   // Validates the given SPIR-V |binary|. Returns true if no issues are found.
   // Otherwise, returns false and communicates issues via the message consumer
   // registered.
+  // Validates for SPIR-V spec rules for the SPIR-V version named in the binary's
+  // header (at word offset 1).  Additionally, if the target environment is a
+  // client API (such as Vulkan 1.1), then validate for that client API version,
+  // to the extent that it is verifiable from data in the binary itself.
   bool Validate(const std::vector<uint32_t>& binary) const;
   // |binary_size| specifies the number of words in |binary|.
   bool Validate(const uint32_t* binary, size_t binary_size) const;
   // Like the previous overload, but takes an options object.
+  // Validates for SPIR-V spec rules for the SPIR-V version named in the binary's
```

Comment:
A little odd to repeat this here, but not in the previous overload.

Reply:
Right. The previous one should say "like the previous one..." as well.  Anyway, I duplicated the text.